Using the latest cached version of the dataset since danavery/urbansound8K couldn't be found on the Hugging Face Hub
Found the latest cached dataset configuration 'default' at /scratch/ptolemy/users/kg1623/.cache/huggingface/datasets/danavery___urbansound8_k/default/0.0.0/8aa9177a0c5a6949ee4ee4b7fcabb01dfd4ae466 (last modified on Sat Apr 26 19:45:24 2025).
/scratch/ptolemy/users/kg1623/projects/deep-learning/audio-classification-using-ViT/venv/lib/python3.10/site-packages/torch/utils/data/dataloader.py:626: UserWarning: This DataLoader will create 8 worker processes in total. Our suggested max number of worker in current system is 1, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(
  0%|          | 0/100 [00:00<?, ?it/s]  1%|          | 1/100 [02:04<3:26:00, 124.85s/it]  2%|▏         | 2/100 [04:07<3:21:49, 123.57s/it]  3%|▎         | 3/100 [06:10<3:19:04, 123.14s/it]  4%|▍         | 4/100 [08:12<3:16:47, 122.99s/it]  5%|▌         | 5/100 [10:15<3:14:27, 122.81s/it]  6%|▌         | 6/100 [12:17<3:12:03, 122.59s/it]  7%|▋         | 7/100 [14:19<3:09:38, 122.35s/it]  8%|▊         | 8/100 [16:21<3:07:41, 122.41s/it]  9%|▉         | 9/100 [18:23<3:05:17, 122.17s/it] 10%|█         | 10/100 [20:25<3:03:11, 122.12s/it] 11%|█         | 11/100 [22:27<3:01:01, 122.03s/it] 12%|█▏        | 12/100 [24:29<2:59:07, 122.13s/it] 13%|█▎        | 13/100 [26:31<2:56:57, 122.04s/it] 14%|█▍        | 14/100 [28:34<2:55:17, 122.30s/it] 15%|█▌        | 15/100 [30:36<2:53:14, 122.28s/it] 16%|█▌        | 16/100 [32:39<2:51:10, 122.27s/it] 17%|█▋        | 17/100 [34:41<2:49:03, 122.22s/it] 18%|█▊        | 18/100 [36:43<2:47:17, 122.41s/it] 19%|█▉        | 19/100 [38:46<2:45:26, 122.55s/it] 20%|██        | 20/100 [40:49<2:43:35, 122.70s/it] 21%|██        | 21/100 [42:52<2:41:37, 122.75s/it] 22%|██▏       | 22/100 [44:56<2:39:55, 123.02s/it] 23%|██▎       | 23/100 [46:59<2:37:47, 122.96s/it] 24%|██▍       | 24/100 [49:01<2:35:27, 122.73s/it] 25%|██▌       | 25/100 [51:03<2:33:17, 122.64s/it] 26%|██▌       | 26/100 [53:06<2:31:10, 122.58s/it] 27%|██▋       | 27/100 [55:08<2:28:57, 122.43s/it] 28%|██▊       | 28/100 [57:10<2:26:41, 122.24s/it] 29%|██▉       | 29/100 [59:12<2:24:39, 122.25s/it] 30%|███       | 30/100 [1:01:14<2:22:36, 122.23s/it] 31%|███       | 31/100 [1:03:16<2:20:25, 122.10s/it] 31%|███       | 31/100 [1:05:18<2:25:21, 126.40s/it]
/scratch/ptolemy/users/kg1623/projects/deep-learning/audio-classification-using-ViT/venv/lib/python3.10/site-packages/torch/__init__.py:2150: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  assert condition, message
